src/blob/core.rs

```diff
@@ -234,8 +257,10 @@ impl Blob {
 
     async fn find_entry<'a>(ents: Entries<'a>, meta: &'a Meta) -> Result<Option<Entry>> {
```

Comment:
Seams this function is not used any more

Reply:
Removed